spec/config-mapper.spec.js

```diff
@@ -114,4 +116,63 @@ describe('Config Mapper', function () {
     writer.hasFiles(hashFive).should.equal(false)
     writer.hasFiles(hashSix).should.equal(true)
   })
+
+  it('should handle missing packagePath by not adding start', function () {
+    sinon.stub(fs, 'existsSync').returns(false)
```

Comment:
Do we have to stub this if the file really doesn't exist?

Reply:
From memory - this was the simplest way to trigger the code - logic is as follows:

- Read the current project's package.json
- Check that the path exists
- If it does - perform the logic to get the `start` element
- If it doesn't - don't add `start` element

Because the logic picks up the package.json from the `kickerd` project it was much simpler to stub the call to get this file so we could pretend it doesn't exist.